### DISCUSSION TITLE
Announcement

### DISCUSSION THREAD

**Discussion by Piercin** [Piercin/Gdevelop3DPhysics#7] [Announcements]
You should no longer use the 3D collision of this Extension. Instead, Use new 3D jump extension, it is smoother. I will try to do other stuff with the extension but I dont think collision is my priority anymore.

**Comment by Yhac10** [comment]
Ok, got it.
Also, where do I get the new extension? I couldn't find it in the repo. Is it built in to Gdevelop?

**Reply by Piercin** [reply]
> Ok, got it.
> Also, where do I get the new extension? I couldn't find it in the repo. Is it built in to Gdevelop?

Its in GDevelop Extensions tab.

**Reply by Yhac10** [reply]
Ok, found it. Also, out of curiosity, what will the future be for this extension?

**Reply by Piercin** [reply]
> Ok, found it. Also, out of curiosity, what will the future be for this extension?

No idea, I haven't worked on it lately because I've been working on other stuff.

**Reply by Yhac10** [reply]
Ok, cool.

**Reply by Piercin** [reply]
> Ok, cool.

If I make anything, It's gonna be more basic stuff, like adding forces in 3D, or camera stuff.

**Reply by Yhac10** [reply]
ok.
